kolibri/content/api.py

```diff
@@ -44,6 +44,8 @@ def title_description_filter(self, queryset, value):
             return exact_match
         # if no exact match, fuzzy search using the stemmed_metaphone field in ContentNode that covers the title and description
         fuzzed_tokens = [fuzz(word) for word in value.split()]
+        if not fuzzed_tokens[0]:
+            return []
```

Comment:
The issue was that if no fuzzed tokens existed for a string such as `wwwwwww`, it would return `[[""]]`